sdk/src/wallet/wallet/operations/stronghold_backup/mod.rs

```diff
@@ -92,9 +93,11 @@ impl Wallet {
             PathBuf::from("wallet.stronghold")
         };
 
+        let stronghold_password = stronghold_password.into();
```

Comment:
the ownership of the String will be transferred to the `Password` type, so `ZeroizeOnDrop` will take care of it.

Reply:
the `impl<T: AsRef<str>> From for Password` can't be used though. I realized it and removed it. Too bad because it kept the convencience what we had before, but then if you pass it a `String s` it would call `s.as_ref().to_owned` which would essentiall create a clone, and `s` would not be zeroized.

Reply:
As a pre-caution I did not impl `From<&str>` or `From<&String`, because that could lead to someone forgetting to zeroize the original manually. That causes some manual clones, but I think safety has a higher priority here.